fastexcel-writer/src/main/java/org/dhatim/fastexcel/Cell.java

```diff
@@ -52,11 +52,13 @@ void write(Writer w, int r, int c) throws IOException {
                 w.append(" s=\"").append(style).append('\"');
             }
             if (value != null && !(value instanceof Formula)) {
-                w.append(" t=\"").append((value instanceof CachedString) ? 's' : (value instanceof Boolean ? 'b' : 'n')).append('\"');
+                w.append(" t=\"").append((value instanceof CachedString) ? "s" : (value instanceof Boolean ? "b" : (value instanceof String ? "inlineStr" : "n"))).append('\"');
```

Comment:
This `?:` chain is getting too long. I suggest introducing a method call here:
```
w.append(" t=\"").append(getCellType(value)).append('\"');
```

Reply:
I added as suggested: https://github.com/dhatim/fastexcel/pull/266/commits/4125b8e0df8495a015cdffae16c9d910a41b1e04